llvm/test/CodeGen/AIE/aie2/schedule/resource/r_wm.mir

```diff
@@ -178,10 +178,10 @@ body:             |
     ; CHECK-LABEL: name: E3_MOVD2_E2_VEXTRACT
     ; CHECK: $r3 = MOV_D2 killed $r1
     ; CHECK-NEXT: NOP
-    ; CHECK-NEXT: $dc0 = VEXTRACT_S16 killed $x1, killed $r16
+    ; CHECK-NEXT: $s0 = VEXTRACT_S16 killed $x1, killed $r16
     ; CHECK-NEXT: NOP
   $r3 = MOV_D2 $r1
-  $dc0 = VEXTRACT_S16 killed $x1, killed $r16
+  $s0 = VEXTRACT_S16 killed $x1, killed $r16
```

Comment:
Check: we changed to an s0 since that does use R_WM, whereas dc0 doesn't

Reply:
exactly!!